cypress/e2e/debug_download.cy.js

```diff
@@ -52,7 +52,7 @@ describe('Debug tab download', function () {
           });
         cy.clearDebugStore();
         cy.get('[data-cy-debug="refresh"]').click();
-        cy.get('[data-cy-debug="tableBody"]').find('tr').should('not.exist');
+        cy.getTableBody().find('tr').should('not.exist');
```

Comment:
Use your function for this:
```suggestion
        cy.functions.assertDebugTableLength(0);
```

Reply:
Added the change.